src/device/rsp/rsp_core.c

```diff
@@ -306,27 +304,45 @@ void do_SP_Task(struct rsp_core* sp)
         rsp.doRspCycles(0xffffffff);
         timed_section_end(TIMED_SECTION_AUDIO);
         sp->regs2[SP_PC_REG] |= save_pc;
+
+        sp_delay_time = 4000;
     }
     else
     {
         sp->regs2[SP_PC_REG] &= 0xfff;
         rsp.doRspCycles(0xffffffff);
         sp->regs2[SP_PC_REG] |= save_pc;
+
+        sp_delay_time = 0;
     }
 
     sp->rsp_task_locked = 0;
     if ((sp->regs[SP_STATUS_REG] & (SP_STATUS_HALT | SP_STATUS_BROKE)) == 0)
```

Comment:
@loganmc10 could you test this https://github.com/Gillou68310/mupen64plus-core/commit/87f9186fb3007f5f4446354442344a7363155219 for regressions?